src/lang/texts.properties

```diff
@@ -25,4 +25,6 @@ blockly.block.url_to_file=Copy path %1 to %2, connection timeout (in ms) %3, rea
 blockly.block.url_to_file.tooltip=It can be used to download an online file.
 blockly.block.write_json=Write the JsonObject %2 inside the file %1
 blockly.block.write_txt=Write following lines inside %1 as a TXT file %2
-blockly.block.write_line=Write %1 in the file %2 and add a new line %3
\ No newline at end of file
+blockly.block.write_line=Write %1 in the file %2 and add a new line %3
+blockly.block.parse_json=Parse json %1
+blockly.block.stringify_json=Stringify json %1
```

Comment:
I try to put those Strings in alphabetic order, so please change their position to follow this logic (as it is easier to find a specific one).

Also, please uppercase `json`, so it is `JSON` everywhere.